src-script/pack-apack-win-linux.js

```diff
@@ -6,12 +6,18 @@ exports.default = async function (context) {
     context.electronPlatformName === 'win32' ||
     context.electronPlatformName === 'linux'
   ) {
-    return scriptUtil.executeCmd({}, 'npx', [
-      'copyfiles',
-      '-V',
-      '-f',
-      path.resolve(context.outDir, '../apack.json'),
-      context.appOutDir
-    ])
+    try {
```

Comment:
I believe our CI has validation for checking this in one of the GitHub actions. So we need not have a try catch.